src/flat-button.jsx

```diff
@@ -255,14 +254,16 @@ const FlatButton = React.createClass({
     const buttonRippleColor = rippleColor || defaultRippleColor;
     const buttonBackgroundColor = backgroundColor || buttonColor;
     const hovered = (this.state.hovered || this.state.isKeyboardFocused) && !disabled;
+    const fontSize = this.state.muiTheme.flatButton.fontSize;
```

Comment:
Might be okay for now, what do you think @oliviertassinari @alitaheri?

If the the theme changes are solid, I'm good merging this as is. I think this file we'll have to revisit to remove `ContextPure` and `getRelevantContextKeys`. That might be a good opportunity to reorganize the assignment of theme variables.